data-catalog-entry-schema.json

```diff
@@ -0,0 +1,263 @@
+            "type": "string"
+          },
+          "name_sci": {
+            "title": "Latin Name (Genus species)",
+            "description": "Linnean Genus species",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "locations": {
+      "title": "Locations",
+      "description": "Info about location(s) of data collection.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "country": {
+            "title": "Country",
+            "description": "",
+            "type": "string",
+            "format": "country"
+          },
+          "region": {
+            "title": "State/Province/Region",
+            "description": "if applicable",
+            "type": "string"
+          },
+          "city": {
+            "title": "City or Town",
+            "description": "nearest city if applicable",
+            "type": "string"
+          },
+          "park": {
+            "title": "Park/Protected Area",
+            "description": "name of park or protected area if applicable",
+            "type": "string"
+          },
+          "field_station": {
+            "title": "Field Station",
+            "description": "name of field station i.e. BCI or Mpala or KOB if applicable",
+            "type": "string"
+          },
+          "lat_long": {
+            "title": "Lat Long",
+            "description": "Latitude (e.g. 9.167) and Longitude (e.g. 47.679) in Decimal Degrees",
+            "type": "string",
+            "format": "lat_long"
+          }
+        }
+      }
+    },
+    "dates": {
+      "title": "Dates",
+      "description": "Select a range of dates where data will be/was collected.",
+      "type": "object",
+      "properties": {
+        "start_date": {
+          "title": "Start Date",
+          "type": "string",
+          "format": "date"
+        },
+        "end_date": {
+          "title": "End Date",
+          "type": "string",
+          "format": "date"
+        }
+      }
+    },
+    "data_type_overview": {
+      "title": "Data Type Overview",
+      "description": "Type(s) of data that will be collected such as behavioral, movement, ecological, tissue samples. Select from a drop down menu, and let us know if you want to make another (or fork and modify changes and ask to merge)",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "examples": [
+          "behavioral",
+          "movement",
+          "vocalizations",
+          "ecological",
+          "climatic",
+          "camera trap",
+          "spatial",
+          "drone imagery",
+          "DNA samples",
+          "tissue samples"
+        ]
+      }
+    },
+    "data_file_type_overview": {
+      "title": "Data File Type Overview",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "examples": [
+          "tsv",
+          "txt",
+          "csv",
+          "mp3"
+        ]
+      }
+    },
```

Comment:
The shiny app has data_type_overview vs. the examples have data_file_type_overview. I included both here. Not sure if both are necessary.